tests/test_lab_dev/test_states/test_states_base.py

```diff
@@ -32,6 +32,7 @@
     Ket,
     Number,
     Vacuum,
+    Sauron,
```

Comment:
Not sure why this is imported here?

Reply:
Also need to include tests in test_states.py

Reply:
oops forgot to take the tests from the other branch 👍